### PR TITLE
persitent localboot & NFS

### PR DESCRIPTION
**Pull request by passowst**
New features: choose between persistent home on local Disk or NFS Server. Benefits for local Boot and maybe caching improvments by PXE Clients on slow NFS Servers

### CODE REVIEW

File: tcos-devices/tcos-devices-2.1/debian/changelog [hunk 1]
```diff
@@ -1,3 +1,20 @@
...
+  * persistent User Home on local parttition
+  *    
+  *     
+  * 
+  *  
+  * 
+  * 
+  *   
+  * 
+  *
```
Comment (steffenhoenig) [inline tcos-devices/tcos-devices-2.1/debian/changelog]: Please fix the many "*" entries and add yourself to the list of developer like
[Stephan Passow]
* Non-maintainer upload.
* persistent ....

File: tcos-devices/tcos-devices-2.1/debian/tcos-devices/schema/device/nfs.xml [hunk 2]
```diff
@@ -1,6 +1,23 @@
...
+          <label lang="en" value="Settings for persistent User Home"/>
+          <label lang="de" value="Einstellung für persistentes User Home"/>
+         	 <choice name="hometype">
+         	  <label lang="de" value="User Home auf" />
+         	  <label lang="en" value="User Home on" />
+               	  <option name="local" value="local" />
+          	  <option name="NFS" value="NFS" />
+            	 </choice>
+</group>          
+          <entry name="partition" value="">
```
Comment (steffenhoenig) [inline tcos-devices/tcos-devices-2.1/debian/tcos-devices/schema/device/nfs.xml]: Why is partition outside of the home group?

Comment (passowst) [inline tcos-devices/tcos-devices-2.1/debian/tcos-devices/schema/device/nfs.xml]: Value won't be saved in apache ds ldap if it is in <group>. 
In the display.xml in tcos-devices it's the same thing..intresting!

File: tcos-devices/tcos-devices-2.1/debian/changelog (hunk 1 above)
Comment (passowst) [inline tcos-devices/tcos-devices-2.1/debian/changelog]: Yes, also needs correct version für major release..

**Pull request closed without merging**